src/Modules/Guilds/Wrappers/ProposalCardWrapper.tsx

```diff
@@ -19,10 +21,19 @@ const ProposalCardWrapper: React.FC<ProposalCardWrapperProps> = ({
   const status = useProposalState(proposal);
   const { withFilters } = useFilter();
   const { options } = useProposalCalls(guildId, proposalId);
-
+  const { address } = useAccount();
+  const { data: proposalVotesOfVoter } = useProposalVotesOfVoter(
+    guildId,
+    proposalId,
+    address
+  );
   return withFilters(
     <ProposalCard
-      proposal={{ ...proposal, id: proposalId }}
+      proposal={{
+        ...proposal,
+        id: proposalId,
+        votesOfVoters: proposalVotesOfVoter,

```

Comment:
Just a naming thing but I see you are using `votesOfVoters` across diff components. I think this should be singular `votesOfVoter` since that is what the contract returns ( voting of a single voter address )

Reply:
Great catch! Will fix immediately!